templates/parameter_table.html

```diff
@@ -1,11 +1,6 @@
 <!-- Parameter/arguments table -->
 <div class="eagleTableDisplay" id="parameterTable" tabindex="-1" role="dialog" aria-labelledby="parameterTableTitle">
     <div class="bottomWindowHeaderWrap" data-bind="click: function(){ParameterTable.resetSelection()}">
-        <!-- ko if: Setting.findValue(Setting.BOTTOM_WINDOW_MODE) === Eagle.BottomWindowMode.GraphConfigAttributesTable -->
-            <h5 class="bottomWindowHeader">
-                Graph Attributes Configuration Table: <span data-bind="html: $root.logicalGraph().getActiveGraphConfig() ? $root.logicalGraph().getActiveGraphConfig()?.name() : ''"></span>
-            </h5>
-        <!-- /ko -->
         <!-- ko if: Setting.findValue(Setting.BOTTOM_WINDOW_MODE) === Eagle.BottomWindowMode.ParameterTable && eagle.selectedNode() != null -->
```

Comment:
Again here, I think maybe you can assume that `BOTTOM_WINDOW_MODE === Eagle.BottomWindowMode.ParameterTable` within this file.